aepsych/models/utils.py

```diff
@@ -237,14 +270,37 @@ class TargetDistancePosteriorTransform(PosteriorTransform):
     def __init__(
         self, target_value: Union[float, Tensor], weights: Optional[Tensor] = None
     ) -> None:
+        """Initialize the TargetDistancePosteriorTransform
+        
+        Args:
+            target_value (Union[float, Tensor]): The target value to transform the posterior to.
+            weights (Tensor, optional): Weights to apply to the target value. Defaults to None.
+        """
```

Comment:
Tensor vs torch.Tensor needs to be consistent, at least within a file.

Reply:
Made all `models` files consistent using `torch.Tensor`. 
Let me know if I missed anything.